README.md

```diff
@@ -706,7 +708,7 @@ So a Local contains a pointer to type T. We can access this pointer using
 
 We can cast from a subtype to a supertype using Local::Cast:
 ```c++
-v8::Local<v8::Number> nr = v8::Local<v8::Number>(v8::Number::New(isolate_, 12));
+v8::Local<v8::Number> nr = v8::Number::New(isolate_, 12);
```

Comment:
Since `v8::Number::New` return `v8::Local<v8::Number>`, we can remove unnecessary type conversion.